po/common/common.pot

```diff
@@ -4865,121 +4865,3 @@ msgstr "Does not contain: %s"
 msgctxt "without_s"
 msgid "Without %s"
 msgstr "Without %s"
```

Comment:
```suggestion
msgstr "Without %s"

msgctxt "owners_p"
msgid "owners"
msgstr ""

msgctxt "owners_s"
msgid "owner"
msgstr ""

msgctxt "org_profile_description"
msgid "You can provide information about your company that will be displayed in your organization profile."
msgstr ""

msgctxt "org_profile_description_2"
msgid "Some of the information like the customer service contact information may also be displayed directly on pages for your products."
msgstr ""

msgctxt "org_name"
msgid "Name"
msgstr "Name"

msgctxt "org_link"
msgid "Link to the official web site"
msgstr ""

msgctxt "org_customer_service"
msgid "Customer service"
msgstr ""

msgctxt "org_customer_service_description"
msgid "Customer service information is public and can be shown on the Open Food Facts web site and apps."
msgstr ""

msgctxt "org_customer_service_note"
msgid "All fields are optional."
msgstr ""

msgctxt "org_commercial_service"
msgid "Commercial service"
msgstr ""

msgctxt "org_commercial_service_description"
msgid "Commercial service information is only shown in the organization profile."
msgstr ""

msgctxt "contact_name"
msgid "Name"
msgstr "Name"

msgctxt "contact_address"
msgid "Address"
msgstr ""

msgctxt "contact_phone"
msgid "Phone number"
msgstr ""

msgctxt "contact_email"
msgid "e-mail address"
msgstr "e-mail address"

msgctxt "contact_link"
msgid "Contact form link"
msgstr ""

msgctxt "contact_info"
msgid "Other information"
msgstr "Other information"

msgctxt "contact_info_note"
msgid "e.g. opening times"
msgstr ""

msgctxt "error_org_does_not_exist"
msgid "The organization was not found."
msgstr ""

msgctxt "error_missing_org_name"
msgid "The organization name is missing."
msgstr ""

msgctxt "edit_org_title"
msgid "Organization profile"
msgstr ""

msgctxt "edit_org_result"
msgid "The organization profile has been updated."
msgstr ""

msgctxt "delete_org"
msgid "Delete the organization"
msgstr ""

msgctxt "official_site"
msgid "Official site"
msgstr ""

msgctxt "contact_form"
msgid "Contact form"
msgstr ""

msgctxt "edit_org_profile"
msgid "Edit your organization profile"
msgstr ""

msgctxt "edit_user_profile"
msgid "Edit your user profile"
msgstr ""

msgctxt "attribute_group_ingredients_analysis"
msgid "Ingredients"
msgstr "Ingredients"

# keep the %s, it will be replaced by an allergen
msgctxt "not_known_s"
msgid "Not known: %s"
msgstr ""

```